pkg/api/aim/projects.go

```diff
@@ -131,7 +147,15 @@ func GetProjectParams(c *fiber.Ctx) error {
 			resp[s] = fiber.Map{}
 		case "metric":
 			var metricKeys []string
-			if tx := database.DB.Model(&database.LatestMetric{}).Distinct().Pluck("Key", &metricKeys); tx.Error != nil {
+			if tx := database.DB.Distinct().Model(
```

Comment:
nitpick: please move the call to `Distinct()` to a similar place as for params and tags

Reply:
sure.

Reply:
done!